source/components/JsxParser.js

```diff
@@ -47,7 +47,7 @@ export default class JsxParser extends Component {
       case 'JSXElement':
         return this.parseElement(expression)
       case 'JSXText':
-        return <Fragment key={randomHash()}>{expression.value || ''}</Fragment>
+        return <div key={randomHash()}>{expression.value || ''}</div>
```

Comment:
Unfortunately, this change would cause the component to fundamentally render wrong/incorrect markup. `JSXText` is things like: `<span>Foo</span>` where `Foo` is the `JSXText`. This change would cause that JSX to render `<span><div>Foo</div></span>` which is an incorrect representation.

I'm going to suggest a couple alternative solutions in the Issue you raised, related to this.

Reply:
Exactly, I have catched such bug now.
Can we just remove the wrapper, like this?
```js
return (expression.value || '')
```
All unit tests are passed in this case.